src/ScreenReaderDriver.h

```diff
@@ -16,7 +16,7 @@ class ScreenReaderDriver {
     hasSpeech(speech),
     hasBraille(braille)
     {}
-  ScreenReaderDriver& operator=(const ScreenReaderDriver&) {}
+  ScreenReaderDriver& operator=(const ScreenReaderDriver&);

```

Comment:
Doesn't this cause a linker error?

I think what you want to do is this:
```ScreenReaderDriver& operator=(const ScreenReaderDriver&) = delete;```
to prevent copying.